### ISSUE TITLE
Viewing of error messages

### ISSUE THREAD

**Issue by lpil** [dermusikman/sonicpi.vim#11]
Hello!

Is it possible to view error messages when a buffer is evaluated unsuccessfully due to an error?

Thanks,
Louis

**Comment by dermusikman** [comment]
Apologies, again, for having gotten to this so late!

Honestly, I only learned enough vimscript and vim internals to satisfy this need; I don't know that vim supports the forwarding of errors from external programs in any sophisticated way. Even if it did, I don't know if this plugin could support all available interfacing programs (are there more than 2 at this point?) without defining some standard, which seems overkill for the niche need. Or, did you mean errors from the sonic-pi-server itself?

I believe that the equivalent emacs script may support such functionality, if you're willing to give evil mode a shot? (I've actually converted to evil-mode emacs since making this plugin. Haven't dabbled in Sonic Pi for a while, unfortunately.)

If you're aware of a solution to scratch this itch, I'm happy to review it in a PR. Closing the issue for now.
